src/main/java/webserver/Request.java

```diff
@@ -3,13 +3,28 @@
 import java.io.BufferedReader;
 import java.io.IOException;
 
+import java.util.HashMap;
+
 public class Request {
+    private static final String EMPTY_SPACE = " ";
```

Comment:
This class seems to have more than one responsibility. What do you think about splitting this up??

Reply:
Should I move the parsing to another class?

Reply:
Yes, I think that'd be good!

Reply:
Just pushed my changes. I moved everything request related to a request package and moved the parsing to it's own class. The Request class only has a builder and getters now.